Yosemite/YosemiteTests/Stores/AppSettingsStoreTests.swift

```diff
@@ -185,4 +185,117 @@ final class AppSettingsStoreTests: XCTestCase {
 
         waitForExpectations(timeout: 2, handler: nil)
     }
+
+    // MARK: - General App Settings
```

Comment:
I don't know if it makes 100% sense, but maybe yes. What do you think about adding a case where `Date` is using a different timezone from the one stored?

Reply:
So I tried this for a while and it looks like it doesn't matter because from the [documentation of `Date`](https://developer.apple.com/documentation/foundation/date):

> A Date value encapsulate a single point in time, independent of any particular calendrical system or time zone. 

There is no timezone in a `Date` so I could not even create a reliable unit test that simulates different time zones. 😞  

As for the plist file, it looks like the `Date` is saved as UTC:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>installationDate</key>
	<date>2020-07-29T18:18:01Z</date>
</dict>
</plist>
```

So I think we are safe here. 🙂